airflow/kubernetes/pod_generator.py

```diff
@@ -506,7 +506,7 @@ def reconcile_containers(base_containers,
             'volume_mounts',
             'mount_path')
         client_container = extend_object_field(base_container, client_container, 'env')
-        client_container = extend_object_field(base_container, client_container, 'env_from')
+        client_container = extend_object_field(base_container, client_container, 'env_from', None)
```

Comment:
Looks good but can you add a test case please in https://github.com/apache/airflow/blob/v1-10-stable/tests/kubernetes/test_pod_generator.py?

Same as https://github.com/apache/airflow/blob/v1-10-stable/tests/kubernetes/test_pod_generator.py#L636-L696 probably